spec/factories/tax_rate_factory.rb

```diff
@@ -3,7 +3,8 @@
 FactoryBot.define do
   factory :tax_rate, class: Spree::TaxRate do
     zone
-    amount { 100.00 }
+    amount { 0.1 }
     tax_category
+    association(:calculator, factory: :default_tax_calculator, strategy: :build)
```

Comment:
Why is this better than `calculator { Calculator::DefaultTax.new }`?

Reply:
They're basically the same, it's just that the `:default_tax_calculator` is added as a named factory as part of these changes in `2-2-stable`, and it starts being referenced in relevant places in tests instead of calling the class directly.

Reply:
Okay, if I understand you correctly than this is following Spree's development and it's easier to follow the same convention.